openml/tasks/functions.py

```diff
@@ -367,79 +367,65 @@ def get_tasks(
     -------
     list
     """
+    if download_data is None:
+        warnings.warn(
+            "`download_data` will default to False starting in 0.16. "
+            "Please set `download_data` explicitly to suppress this warning.",
+            stacklevel=1,
+        )
+        download_data = True
+
+    if download_qualities is None:
+        warnings.warn(
+            "`download_qualities` will default to False starting in 0.16. "
+            "Please set `download_qualities` explicitly to suppress this warning.",
+            stacklevel=1,
+        )
+        download_qualities = True
+
     tasks = []
     for task_id in task_ids:
-        tasks.append(get_task(task_id, download_data, download_qualities))
+        tasks.append(
+            get_task(task_id, download_data=download_data, download_qualities=download_qualities)
+        )
     return tasks
 
 
 @openml.utils.thread_safe_if_oslo_installed
 def get_task(
     task_id: int,
-    *dataset_args: Any,
-    download_splits: bool | None = None,
+    download_splits: bool = False,  # noqa: FBT001, FBT002
     **get_dataset_kwargs: Any,
 ) -> OpenMLTask:
```

Comment:
Not recommending to do anything but just pointing out this is technically a backwards breaking change, given the *args was removed.

Reply:
While no deprecation warning was given, it was included in the docs:

> The args are only present for backwards compatibility and will be removed from version 0.15.0 onwards.

Since those `*args` can still be set ( through `**kwargs`) and it's pretty niche, I chose to let it slide and not add a more explicit deprecation cycle after discovering this.